cmd/ocm-backplane/cloud/assume.go

```diff
@@ -0,0 +1,140 @@
+package cloud
+
+import (
+	"context"
+	"encoding/json"
+	"fmt"
+	"github.com/aws/aws-sdk-go-v2/aws"
+	"github.com/aws/aws-sdk-go-v2/credentials"
+	"github.com/aws/aws-sdk-go-v2/service/sts"
+	"github.com/openshift/backplane-cli/pkg/awsUtil"
+	"github.com/openshift/backplane-cli/pkg/cli/config"
+	"github.com/openshift/backplane-cli/pkg/utils"
+	"github.com/spf13/cobra"
+	"io"
+)
+
+const (
+	DefaultRoleArn = "arn:aws:iam::922711891673:role/SRE-Support-Role"
```

Comment:
Is it a good idea to put our aws account id to public? 
Should be better if we read this as environment, and we can read it from the code?
Or we can also put this in to the config file, and we can use it directly. 
We just need to add another field to the config file https://github.com/openshift/backplane-cli/blob/main/cmd/ocm-backplane/config/config.go#L7

Reply:
I had the same concerns, but @jharrington22 reassured us that it was fine to expose the account ID since the role can't be assumed without an SSO account anyway.

https://redhat-internal.slack.com/archives/C058T1YEDPV/p1684870169269229?thread_ts=1684868454.749649&cid=C058T1YEDPV